src/applications/personalization/profile/util/direct-deposit.js

```diff
@@ -14,44 +15,42 @@ export class DirectDepositClient {
     return this.useLighthouseEndpoint ? this.#LH_ENDPOINT : this.#PPIU_ENDPOINT;
   }
 
-  generateApiRequestOptions(payload) {
-    set(payload, 'financialInstitutionName', 'Hidden form field');
+  generateApiRequestOptions(fields) {
+    // The PPIU endpoint REQUIRES a financialInstitutionName field, but the
+    // Lighthouse endpoint does not. We set a dummy value here to avoid
+    // validation errors aka 500s from vets-api
+    set(fields, 'financialInstitutionName', 'Hidden form field');
```

Comment:
This is the import part that caused updates to Direct deposit to fail before, so I added some comments around why it is needed.